README.md

```diff
@@ -1,10 +1,5 @@
-Coderwall SVG [![Donate on Gittip](http://badgr.co/gittip/twolfson.png)](https://www.gittip.com/twolfson/)
+Coderwall SVG [![Donate on Gittip](https://img.shields.io/gratipay/twolfson.svg)](https://gratipay.com/~twolfson/)
```

Comment:
This pattern is very out of date with how I typically link to gratipay. Please leave this in its original form and I will update it accordingly in another PR/release